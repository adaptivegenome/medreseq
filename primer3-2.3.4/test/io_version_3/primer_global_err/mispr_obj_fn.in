PRIMER_SEQUENCE_ID=mispr_lib_not_defined
PRIMER_WT_REP_SIM=1
SEQUENCE=CAGCTGATCGATCGATCGATCGTAGCTAGCTAGCTAGCTAGCTGATCGATCGATGCTAGCTAGCTAGCTAGTCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTAGCTGACTGATCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCT
=
