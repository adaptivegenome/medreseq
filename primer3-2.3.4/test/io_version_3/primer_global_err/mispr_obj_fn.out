PRIMER_SEQUENCE_ID=mispr_lib_not_defined
PRIMER_WT_REP_SIM=1
SEQUENCE=CAGCTGATCGATCGATCGATCGTAGCTAGCTAGCTAGCTAGCTGATCGATCGATGCTAGCTAGCTAGCTAGTCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCTAGCTAGCTGACTGATCGATCGATGCTAGCTAGCTAGCTAGCTGATCGATCGTAGCTAGCT
PRIMER_ERROR=Mispriming score is part of objective function, but mispriming library is not defined
=
